github/git_blobs.go

```diff
@@ -38,6 +39,23 @@ func (s *GitService) GetBlob(ctx context.Context, owner string, repo string, sha
 	return blob, resp, err
 }
 
+// GetBlobRaw fetches a blob's contents from a repo.
+// Unlike GetBlob, it returns the raw bytes rather than the base64-encoded data.
+func (s *GitService) GetBlobRaw(ctx context.Context, owner, repo, sha1 string) ([]byte, error) {
```

Comment:
This method doesn't return `*Response`. Is there a specific reason you left it out?

Unless there's a reason for it, I believe it's needed for consistency and for callers to be able to find out more details about the response.

Reply:
I don't know what one would do with the Response apart from what is in the []byte and error return values, but sure.

Reply:
It might be helpful, for example, to log the response status code from GitHub API in case an error happens.

It's mostly for consistency. All other endpoints return it, so users would expect to be able to access the `*github.Response` struct in this case too.